autoemulate/experimental/data/utils.py

```diff
@@ -17,8 +17,9 @@ class ConversionMixin:
     Mixin class to convert input data to pytorch Datasets and DataLoaders.
     """
 
+    @classmethod
```

Comment:
Updated to `@classmethod` so that they can be called independently of an instance.